src/test/java/framework/DarkSkyLandingPage.java

```diff
@@ -0,0 +1,114 @@
+package framework;
+
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import ru.yandex.qatools.allure.annotations.Step;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+public class DarkSkyLandingPage extends BasePage {
+
+    private By twoHourIncrement = By.xpath("//div[@id='timeline']//div[@class='hours']//span[3]");
```

Comment:
Good attempt but you need to have a single locator instead one for each time slot